andela/mrm_api
# Introducing bugsnag to converge backend

#### Description

In order to help with troubleshooting issues on staging and productions, we would like to use Bugsnag to log any issues on the environments. Bugsnag is already configured, we just need to use it. In order to improve performance, use Celery to handle tasks so logging is non-blocking.

#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
 - Check the `code climate` report to see if any file from the fixtures folder violates the `file-line-limit`
- create an account on bugsnag and put the secret key in .env, then fire any request which creates the error. You should receive an error notification in the bugsnag dashboard and on GraphQl both.
 - Follow the instructions on the `readme.md` file on running tests. Check to see if tests run successfully 

#### Checklist

#### JIRA
[CV3-92](https://andela-apprenticeship.atlassian.net/browse/CV3-92)
